content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,41 @@
+
+| Setting            | Description                                                         |
+|--------------------|---------------------------------------------------------------------|
+| project-name       | Name of the current project.                                        |
+| project-cache-path | Full path to the project cache folder.                              |
+| project-path       | Supplies the path to the project that the AP should use (required). | 
+| engine-path        | Supplies the engine path to the engine.                             |
+
+
+## Advanced Settings
+You can also control the behavior using the following command line options:
+
+| Setting                         | Description                                                                                                                                                                                                 | Notes                                    |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------|
+| waitOnLaunch                    | Briefly pauses Asset Processor during initialization. Useful if you want to attach a debugger.                                                                                                              | Defaults to false for AP, true for batch |
+| zeroAnalysisMode                | Enables using file modification time when examining source assets for processing.                                                                                                                           |                                          | 
+| enableQueryLogging              | Enables logging database queries.                                                                                                                                                                           |                                          |
+| dependencyScanPattern (dsp)     | Scans assets that match the given pattern for missing product dependencies.                                                                                                                                 |                                          |
+| fileDependencyScanPattern (fsp) | Used with `dependencyScanPattern` to farther filter the scan                                                                                                                                                |                                          |
+| additionalScanFolders.          | Used with `dependencyScanPattern` to farther filter the scan.                                                                                                                                               |                                          |
```

Comment:
There probably shouldn't be a period included at the end of the argument name

Reply:
Good call, will fix.